don't set contentType if no data is sent

This makes CORS easier since there won't always be a
preflight request.
Index: swagger-ui/src/main/coffeescript/view/OperationView.coffee
===================================================================
--- swagger-ui.orig/src/main/coffeescript/view/OperationView.coffee
+++ swagger-ui/src/main/coffeescript/view/OperationView.coffee
@@ -153,6 +153,9 @@ class OperationView extends Backbone.Vie
       if paramContentTypeField
         obj.contentType = paramContentTypeField
 
+      if not obj.data
+        obj.contentType = false
+
       responseContentTypeField = $('.content > .content-type > div > select[name=contentType]', $(@el)).val()
       if responseContentTypeField
         obj.headers = if obj.headers? then obj.headers else {}
